tests/unit/dags/test_pwdb_classifiers_dag.py

```diff
@@ -1,8 +1,8 @@
-from sem_covid.entrypoints.ml_dags.pwdb_classifiers_dag import DAG_NAME
+from sem_covid.entrypoints.ml_dags.classifiers_dag import classifier_dag_name
```

Comment:
Shouldn't we delete all the modules and tests that concern `pwdb_classifier` etc. ?